docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,341 @@
+# SoftAP
+
+SoftAP means Software Access Point for local area connections without internet access.
+
+The main features of the SoftAP API includes:
+
+- Enabling and Disabling SoftAP
+
+ You can turn SoftAP on or off.
+
+
+- Managing SoftAP
+
+ You can manage the SoftAP configuration settings such as Service Set Identifier (SSID) and passphrase.
```

Comment:
I think there is an alignment issue. "You can" is one space to the right side.
Please verify and update.